client/state/happychat/test/selectors.js

```diff
@@ -92,49 +92,33 @@ describe( 'selectors', () => {
 			HAPPYCHAT_CHAT_STATUS_NEW,
 		];
 
-		it( 'should return false if Happychat is unavailable', () => {
+		it( 'should return false if Happychat is not connected', () => {
```

Comment:
> Also I'm unsure if `connectionStatus==="connected"` is the same as "Is user currently chatting"

It's not — `chatStatus===HAPPYCHAT_CHAT_STATUS_ASSIGNED` is what tells us "user is currently chatting". Users should be able to send messages outside of an ongoing conversation too, namely when the status is `_NEW`, `_ASSIGNING`, and `_CLOSED` — and there are already tests to check all of this.